openmc/tallies.py

```diff
@@ -2522,6 +2522,68 @@ def __neg__(self):
         new_tally = self * -1
         return new_tally
 
+    def get_data_slice_where(self, basis: str='xy', slice_value: float=None):
```

Comment:
I'd indicate in the name of the method that this is specific to slicing data related to a  MeshFilter somehow.

Reply:
I've tried another name but can't find one with mesh filter that I like, leave this with me to mull over

Reply:
tried adding ```from_mesh``` to the end of the name